invenio_records_resources/services/file_metadata_service.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Record Service API."""
```

Comment:
minor: file naming again. If we are in the `service` folder IMHO there is no need to put `service` in the name. See views, resources, etc. they all are `records` etc. no `records_view` or `records_resource`. (Just the file name to make them shorter which helps on imports).

Reply:
Was trying to see if it made it easier to find the file in my text editor, but it doesn't make a difference really. Renaming!